at_end2end_test/config/config.yaml

```diff
@@ -5,5 +5,9 @@ root_server:
 
 # cicd atSign details
 atSign:
-  firstAtSign: '@cicd1'
-  secondAtSign: '@cicd2'
+  # @ce2e1 is running latest at_secondary_server trunk
+  # @ce2e3 is running the latest canary release
+  # See https://github.com/atsign-foundation/at_client_sdk/issues/447#issuecomment-1075290415 for
+  # full details.
+  firstAtSign: '@ce2e1'
+  secondAtSign: '@ce2e3'
```

Comment:
This atsign is running on the same VM as firstAtSign, which isn't necessarily a problem, but past convention has been to test between VMs

Reply:
OK ... chose 1 & 3 to get different secondary versions. What would you suggest the best pairings would be? How about 1 & 4, and 3 & 2?

Reply:
Yes, 1&4 and 3&2 will give a cross test of old vs new. I'm not quite sure why we're not using the same 3 way matrix as at_server, but this is what @kalluriramkumar asked for.

Reply:
Paired ce2e1 and ce2e4